cs3client/cs3client.py

```diff
@@ -47,13 +47,13 @@ def __init__(self, config: ConfigParser, config_category: str, log: logging.Logg
         self._gateway: cs3gw_grpc.GatewayAPIStub = cs3gw_grpc.GatewayAPIStub(self.channel)
         self._status_code_handler: StatusCodeHandler = StatusCodeHandler(self._log, self._config)
         self.auth: Auth = Auth(self._config, self._log, self._gateway)
```

Comment:
This should be removed. Is the client's integrator who is responsible for creating the Auth object. Otherwise, this defeats the purpose of having a single client with multiple auths.

Reply:
This means that your Auth class should accept a client as a parameter and then extract the things it needs to run internally.